### ISSUE TITLE
Rule to prevent using xpath

### ISSUE THREAD

**Issue by wlsf82** [cypress-io/eslint-plugin-cypress#218]
A mentee of mine (@RenanDezotti) asked in my mentorship Slack group if it would be possible to create a rule using `eslint-plugin-cypress` to prevent users from using xpath selectors with Cypress.

I told him this wasn't possible but would be a great addition to the plugin.

So here's the idea.

I hope someone can pick it up and implement it.

**Comment by jennifer-shehane** [comment]
I agree with this sentiment. There are a lot more people than you think using xpath.

**Comment by wlsf82** [comment]
> I agree with this sentiment. There are a lot more people than you think using xpath. 

I can imagine.

**Comment by AndradeJV** [comment]
I agree with you Walmyr and I think should interesting this rule!

**Comment by MikeMcC399** [comment]
- #247 has been submitted to implement this request

**Comment by duranbe** [comment]
Released in 4.2.0 😄 should  be good to close
